seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/MultiTableMetricsIT.java

```diff
@@ -178,6 +196,30 @@ public void multiTableMetrics() {
                                             && Double.parseDouble(
                                                             response.path(
                                                                     "metrics.TableSinkWriteBytesPerSeconds.'fake.public.table2'"))
+                                                    > 0
+                                            && Double.parseDouble(
+                                                            response.path(
+                                                                    "metrics.TransformCount.'fake.table1'.'fake1->fake3'"))
```

Comment:
Not all sql transform contains `plugin_input` or `plugin_output`. So I think we can not put it into metrics name. How about use `metrics.TransformCount.Transform1.'fake.table1'`?